src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -418,19 +418,19 @@ public void testRemoveNonAscii() {
 
     @Test
     public void testJoin() {
```

Comment:
You can remove all these tests because `String.join` is no longer our own code (i.e. only the tests taking in list of integers should remain).